### ISSUE TITLE
64 bit requirement

### ISSUE THREAD

**Issue by yadavkohi** [hiteshsondhi88/ffmpeg-android-java#316]
As per the google guidelines we need 64 bit architecture, but the existing code is having 32-bit architecture. please update the architecture.

reference of the guideline : 
https://developer.android.com/distribute/best-practices/develop/64-bit

**Comment by MereFable** [comment]
> As per the google guidelines we need 64 bit architecture, but the existing code is having 32-bit architecture. please update the architecture.
> 
> reference of the guideline :
> https://developer.android.com/distribute/best-practices/develop/64-bit

Not sure why you think this project will be updated to meet these requirements considering the last commit was Apr 21, 2016, almost 3 years ago.

**Comment by yadavkohi** [comment]
Actually google has update his guidelines for app store release. According to that app must be supported 64 bit architecture from 1-August-2019, but this source code support 32 bit, so this app will not run on device which is having 64 bit architecture from 1-aug-2019.

**Comment by SomeGuyOnGH** [comment]
> Actually google has update his guidelines for app store release. According to that app must be supported 64 bit architecture from 1-August-2019, but this source code support 32 bit, so this app will not run on device which is having 64 bit architecture from 1-aug-2019.

Yes, I am aware of that, but this project itself hasn't had a commit in 3 years. I don't think they're going to be making a 64 bit version. You would likely be better off compiling your own ffmpeg and then create your own wrapper or find a different version that offers 64 bit.

**Comment by MereFable** [comment]
> Actually google has update his guidelines for app store release. According to that app must be supported 64 bit architecture from 1-August-2019, but this source code support 32 bit, so this app will not run on device which is having 64 bit architecture from 1-aug-2019.

I am aware of this, but what I was saying is, this repository hasn't had a commit in 3 years. The chances they are going to come in and add 64 bit support to this package is very very slim. They don't appear to even respond to any issues anymore. You would be better off finding a different package that does support 64 bit already or dissect this one and add in your own 64 bit support.

**Comment by KishanViramgama** [comment]
see this library i think useful it

[EpMedia](https://github.com/yangjie10930/EpMedia)

**Comment by munix** [comment]
One thing I don't understand is that it boasts of releasing great code but doesn't put executables to recompile ffmpeg and that we can give it 64-bit compatibility or improve the project. A pity

**Comment by softwrengr** [comment]
@MereFable  do you have any sollution please guide us ?
